administrator/components/com_associations/associations.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="component" version="3.6" method="upgrade">
+    <name>com_associations</name>
+    <author>Joomla! Project</author>
+    <creationDate>May 2006</creationDate>
+    <copyright>(C) 2005 - 2016 Open Source Matters. All rights reserved.</copyright>
+    <license>GNU General Public License version 2 or later; see LICENSE.txt</license>
+    <authorEmail>admin@joomla.org</authorEmail>
+    <authorUrl>www.joomla.org</authorUrl>
+    <version>3.0.0</version>
+    <description>COM_ASSOCIATIONS_XML_DESCRIPTION</description>
+    <administration>
+        <files folder="admin">
+            <filename>access.xml</filename>
+            <filename>config.xml</filename>
+            <filename>associations.php</filename>
+            <filename>controller.php</filename>
+            <folder>controllers</folder>
+            <folder>elements</folder>
+            <folder>helpers</folder>
+            <folder>models</folder>
+            <folder>views</folder>
```

Comment:
Please add a "language" folder which will contain the strings. With this, we can "discover" the new component and have language.